.github/workflows/e2e-suite.yml

```diff
@@ -112,7 +122,13 @@ jobs:
         run: |
           timestamp=$(date +'%Y%m%d%H%M')
           report_filename="${timestamp}_cli_test_report.xml"
-          make testint TEST_ARGS="--junitxml=${report_filename}"
+          if [ -n "$OPENAPI_SPEC_URL" ]; then
+            echo "Running tests with OpenAPI spec from: $OPENAPI_SPEC_URL"
+            make testint TEST_ARGS="--openapi-spec-url=${OPENAPI_SPEC_URL} --junitxml=${report_filename}"
```

Comment:
I think the `--openapi-spec-url=...` snippet can be removed here since it's handled in the install above